src/js/dqa-scores.jsx

```diff
@@ -0,0 +1,357 @@
+var React = require('react'),
+    _ = require('underscore');
+
+var FixedDataTable = require('fixed-data-table'),
+    Column = FixedDataTable.Column,
+    Table = FixedDataTable.Table;
+
+var isColumnResizing;
+
+var SortTypes = {
+    ASC: 'ASC',
+    DESC: 'DESC',
+};
+
+var Status = {
+    'new': {
+        sortOrder: 0,
+        desc: 'The issue has been identified by the Data Coordinating Center.',
+    },
+    'under review': {
+        sortOrder: 1,
+        desc: 'The site has acknowledged that it needs to investigate the issue.',
+    },
+    'solution proposed': {
+        sortOrder: 2,
+        desc: ('The site has investigated the issue, and ' +
+               'a consensus about the solution has been reached; ' +
+               'the site has agreed to fix the issue in the next round.'),
+    },
+    'default': {
+        sortOrder: 10,
+    },
+    'persistent': {
+        sortOrder: 20,
+        label: 'inherent',
+        desc: ('The issue has been investigated, ' +
+               'and it was concluded that it cannot be fixed'),
+    },
+    'withdrawn': {
+        sortOrder: 30,
+        desc: 'False alarm.  An investigation has concluded this to be a non-issue.',
+    },
+    'non-issue': {
+        sortOrder: 31,
+        desc: 'False alarm.  An investigation has concluded this to be a non-issue.',
+    },
+    'closed': {
+        sortOrder: 32,
+        desc: '',
+    },
+};
+        
+var DQAScores = React.createClass({
+    propTypes: {
+        columnWidth: React.PropTypes.number,
+        data: React.PropTypes.object,
+    },
+
+    getDefaultProps: function() {
+        return {
+            columnWidth: 120,
+            data: {},
+        };
+    },
+
+    getInitialState: function() {
+        return {
+            columnWidths: {},
+            isCollapsed: false,
+            rows: {},
+            sortBy: null,
+            sortDir: null,
+            statusList: [],
+            tableWidth: 500,
+        };
+    },
+
+    _initRows: function(props) {
+        var i;
```

Comment:
It is preferred to key variables that are only applicable to a specific block next the block. When I was looking at line 89 then 98-109, I didn't see the `var ...` declaration which made me think there was a memory leak (undeclared variable). Even though the `i` variable is reused, it would more clear in loops to do `if (var i in statusList)` rather than defining it externally.